src/common/interflop.h

```diff
@@ -21,6 +21,29 @@ enum FCMP_PREDICATE {
   FCMP_TRUE,
 };
 
+/* Enumeration of types managed by function instrumentation */
+enum FTYPES { FFLOAT, FDOUBLE, FTYPES_END };
+
+typedef struct interflop_function_info {
+  // Indicate the identifier of the function
+  char id[500];
+  // Indicate if the function is from library
+  short isLibraryFunction;
+  // Indicate if the function is intrinsic
+  short isIntrinsicFunction;
+  // Indicate if the function use double
+  short useFloat;
+  // Indicate if the function use float
+  short useDouble;
+} interflop_function_info_t;
+
+/* Verificarlo call stack */
+typedef struct interflop_function_stack {
+  interflop_function_info_t **array;
+  long int capacity;
+  long int top;
```

Comment:
It's unfortunate we still need both capacity and top. 
In previous discussions I though we discussed removing top, and always passing a interflop_function_stack were 0 is the top element, 1 the second, 2 the third, etc.

Reply:
Capacity is no longer in the structure